pkgs/by-name/pl/plus-jakarta-sans/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "plus-jakarta-sans";
+  version = "2.7.1";
+
+  src = fetchzip {
+    url = "https://github.com/tokotype/PlusJakartaSans/releases/download/${version}/PlusJakartaSans-${version}.zip";
+    stripRoot = false;
+    hash = "sha256-+ghkQ4/wxou3FB68ceCwBeGrSCaHJhb16MybvN6mCSc=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/truetype
+    cp PlusJakartaSans-$version/*/PlusJakartaSans*.ttf $out/share/fonts/truetype
```

it's more common to use GNU install in Nixpkgs
```suggestion
    install -Dm644 PlusJakartaSans-$version/*/PlusJakartaSans*.ttf -t $out/share/fonts/truetype
```